lib/SILOptimizer/Utils/SILInliner.cpp

```diff
@@ -424,17 +424,39 @@ SILInlineCloner::cloneInline(ArrayRef<SILValue> AppliedArgs) {
   SmallVector<SILValue, 4> entryArgs;
   entryArgs.reserve(AppliedArgs.size());
   SmallBitVector borrowedArgs(AppliedArgs.size());
+  SmallBitVector copiedArgs(AppliedArgs.size());
 
   auto calleeConv = getCalleeFunction()->getConventions();
   for (auto p : llvm::enumerate(AppliedArgs)) {
     SILValue callArg = p.value();
     unsigned idx = p.index();
-    // Insert begin/end borrow for guaranteed arguments.
-    if (idx >= calleeConv.getSILArgIndexOfFirstParam() &&
-        calleeConv.getParamInfoForSILArg(idx).isGuaranteed()) {
-      if (SILValue newValue = borrowFunctionArgument(callArg, Apply)) {
-        callArg = newValue;
+    if (idx >= calleeConv.getSILArgIndexOfFirstParam()) {
+      if (Apply.getFunction()->hasOwnership() &&
+          Apply.getFunction()
```

Comment:
To make the code readable, I would create a local variable at the top of this function called `EnableLexicalLifetimes`

Reply:
Fixed!